src/components/List.js

```diff
@@ -26,10 +28,44 @@ const List = ({ token }) => {
   };
 
   const handleCheckbox = async (event) => {
-    const currentDate = Date.now(); // changed to using Date.now(), time given in milliseconds
+    const currentDate = Date.now(); // time given in milliseconds
     const queryCollection = await db.collection(token).doc(event.target.id);
+    // targetDoc is our current snapshot of the doc
+    const targetDoc = docs.filter((doc) => doc.id === queryCollection.id);
+    const previouslyPurchasedDate = targetDoc[0].lastPurchased;
+    const latestInterval = currentDate - previouslyPurchasedDate;
+    // If item has not been purchased set to 1, else increment by 1
+    const numberOfPurchases = !targetDoc[0].numberOfPurchases
+      ? 1
+      : (targetDoc[0].numberOfPurchases += 1);
+    const estimate = calculateEstimate(
+      targetDoc[0].timeFrame,
+      latestInterval,
+      numberOfPurchases,
+    );
+
+    // FOR THE DEMO
+    // ------------------------------------------------------------------------------------
+    // update day of demo
+    // const oneWeekAgo = getTime(new Date(2021, 0, 26, 11, 30, 30));
+    // oneWeekAgo simulates the last date of purchase for the item
+    // console.log("Date a week ago: ", toDate(new Date(oneWeekAgo)));
+    // const demoLatestInterval = currentDate - oneWeekAgo;
+    // const exampleNumberOfPurchases = 7;
+    // const demoEstimate = calculateEstimate(
+    //   targetDoc[0].timeFrame,
+    //   demoLatestInterval,
+    //   exampleNumberOfPurchases,
+    // );
+    // demo estimate tells how long from purchase date until we're expected to buy it again
+    // console.log('Demo estimate :', addMilliseconds(previouslyPurchasedDate, demoEstimate));
+    // ------------------------------------------------------------------------------------
+    // END OF DEMO
```

Comment:
This code is specifically for our demo on Sunday. We will install the date-fns dependency to demo the times in the console. However, this is not necessary for the functionality and will be removed before merging.

Reply:
Nice! Love the idea of having something like this ready to help out with the demo. 
One thought that I had is that you could create a demo branch where you would keep this code, then you could merge the intended code changes before the demo without merging this demo code and also have something socked away that you can use for demo purposes.